src/js/pages/jobs/JobDetailPageContainer.js

```diff
@@ -18,170 +14,112 @@ export const DIALOGS = {
   DESTROY: "destroy"
 };
 
-const LoadingScreen = function({ jobTree }) {
+const LoadingScreen = function() {
   return (
     <Page>
-      <Page.Header breadcrumbs={<JobsBreadcrumbs tree={jobTree} />} />
+      <Page.Header breadcrumbs={[]} />
       <Loader />
     </Page>
   );
 };
 
-const ErrorScreen = function({ jobTree }) {
+const ErrorScreen = function() {
```

Comment:
(not important right now) I'd love to see these moved to its own files, reason is, that I have exaclty the same code and then we would be able to reuse stuff. now its all hidden in this big one file. :/